include/seastar/http/reply.hh

```diff
@@ -111,6 +111,38 @@ struct reply {
         network_connect_timeout = 599, //!< network_connect_timeout
     } _status;
 
+    /**
+     * HTTP status classes
+     * See https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
+     *
+     * 1xx: Informational - Request received, continuing process
+     * 2xx: Success - The action was successfully received, understood, and accepted
+     * 3xx: Redirection - Further action must be taken in order to complete the request
+     * 4xx: Client Error - The request contains bad syntax or cannot be fulfilled
+     * 5xx: Server Error - The server failed to fulfill an apparently valid request
+     */
+    enum class status_class : uint8_t {
+        informational=1,
```

Comment:
could you add space around `=` for better readability?

Reply:
sure